presto-native-execution/presto_cpp/main/TaskResource.cpp

```diff
@@ -188,11 +188,6 @@ proxygen::RequestHandler* TaskResource::acknowledgeResults(
                 http::sendOkResponse(downstream);
               }
             })
-            .thenError(
-                folly::tag_t<velox::VeloxException>{},
-                [downstream](auto&& e) {
-                  http::sendErrorResponse(downstream, e.what());
```

Comment:
The VeloxException::what() is not default std::exception::what(). Removing this block might cause error info loss. Could you check VeloxException.h to verify? Thanks